### DISCUSSION TITLE
MagPi Magazine Article on BirdNET-Pi

### DISCUSSION THREAD

**Discussion by CaiusX** [mcguirepr89/BirdNET-Pi#415] [General]
Hi All

The official Raspberry-Pi magazine, Magpi has published a nice article on @mcguirepr89 's fantastic system - 

You can download the free PDF of the magazine [HERE](https://magpi.raspberrypi.com/issues/119/pdf).

Please consider donating to keep this mag going, and, of course, buying Patrick some coffee by sponsoring on Github [HERE](https://github.com/mcguirepr89)

**Comment by nilspupils** [comment]
Congratulations!

**Comment by ehpersonal38** [comment]
So cool!! Good job Patrick 🎉

**Comment by mcguirepr89** [comment]
This is awesome!! I'm bummed that the names from this repo that I mentioned as invaluable help didn't make it into the article:

I was hoping the article would mention these folks by name and contributions:
- @CaiusX Data visualization and testing
- @ehpersonal38 web GUI overhauling, repo maintenance help, feature hunter and deliverer
- @DD4WH ornithological/biological insight and DIY microphone design
- @nilspupils reliable testing, feedback and unending support and enthusiasm
- Bill Powers @ [PixCams](https://pixcams.com) for the awesome build and photos

My very best regards to you all!! And a thanks to Rosie if she ever sees this!!
Patrick

**Comment by ade-secrutiny** [comment]
I'm a direct migrant from that article to this project - which I'm having great fun with!

**Reply by CaiusX** [reply]
Welcome to the flock!
